Gemfile

```diff
@@ -3,11 +3,10 @@ gemspec
 
 gem "rake"
 
-# Lock standard to a particular version, esp. cause it's still 0.x.x according to Semver
-gem "standard", "0.4.6"
+gem "standard", "~> 1.0"
 
 gem "rspec", "~> 3.9"
 
 gem "yard"
 
-gem "climate_control"
+gem "climate_control", "0.2.0"
```

Comment:
Should we just remove support for ruby 2.4? It's been EOL for over 3 years now

Reply:
I just updated sidekiq-encrypted_args with a spec helper to replace climate_control that we could port over. I'm all for deprecating 2.4 support, but probably need at least a minor point release to justify it.

Reply:
does that mean it's time for the world to meet `simple_env_control` then 😂

Kidding aside, I'd also just _generally_ worry about anyone still trying to do encryption/linking against openSSL with a version of ruby that old